### ISSUE TITLE
از کار افتادن تانل وارپ بعد از آپدیت 1.4

### ISSUE THREAD

**Issue by techjoyment** [hiddify/hiddify-next#955]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

یک کانفیگ داشتیم که به راحتی با پینگ پایین بعد از سوئیچ تانل وارپ (گذر پراکسی از داخل وارپ) متصل میشد. بعد از آپدیت به 1.4 دیگه وصل نمیشه.

### Minimal Reproducible Example (MRE)

یک کانفیگ اد کنید.
سوئیچ وارپ رو بزنید.
ترجیحا وارپ خود هیدیفای رو هم اضافه کنید به کانفیگ.

### Version

1.4.0

### Platform/OS

Windows

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by papampi** [comment]
Same problem ...
Warp works ok in profiles via WarpOnWarp or even VlessOnWarp, but Warp from config setting to set "Detour Proxies Through Warp" always gives timeout.

**Comment by LitenBuzzTh** [comment]
> Same problem ... Warp works ok in profiles via WarpOnWarp or even VlessOnWarp, but Warp from config setting to set "Detour Proxies Through Warp" always gives timeout.

exactly like mine

**Comment by danialkeimasi** [comment]
Same here

**Comment by danialkeimasi** [comment]
Tested on GNU/Linux, it has been fixed for me.

**Comment by danialkeimasi** [comment]
On Android, it gets stuck on "Connecting" when WARP is enabled. WARP tunnel worked fine in version `1.1.1` on the same network. I have also re-generate WARP config and nothing changed.

**Comment by danialkeimasi** [comment]
Tested on GNU/Linux with "set system proxy" mode, it has been fixed.

On Android with VPN mode, it gets stuck on "Connecting" when WARP is enabled. WARP tunnel worked fine in version `1.1.1` on the same network. I have also re-generate WARP config and nothing changed.

**Comment by danialkeimasi** [comment]
On GNU/Linux connection is unstable and remain connecting although it is connected. also got this error:

![image](https://github.com/user-attachments/assets/5b9c5f21-de19-41e2-9359-dbdf9f138e68)

@titancomputer

**Comment by LitenBuzzTh** [comment]
> Tested on GNU/Linux with "set system proxy" mode, it has been fixed.
> 
> On Android with VPN mode, it gets stuck on "Connecting" when WARP is enabled. WARP tunnel worked fine in version `1.1.1` on the same network. I have also re-generate WARP config and nothing changed.

exactly. on android, in vpn mode, only the 1.1.1 version seems to work (not always, since requires many cycles on connect/disconnect)
